### PR TITLE
Hughsie/vendor from parent

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/amd-gpu/fu-amd-gpu-device.c [hunk 1]
```diff
@@ -94,8 +94,6 @@ fu_amd_gpu_device_probe(FuDevice *device, GError **error)
 		fu_device_set_logical_id(device, "rom");
 		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_CAN_VERIFY_IMAGE);
 		fu_udev_device_add_open_flag(FU_UDEV_DEVICE(device), FU_IO_CHANNEL_OPEN_FLAG_READ);
-		fu_udev_device_add_flag(FU_UDEV_DEVICE(device),
-					FU_UDEV_DEVICE_FLAG_VENDOR_FROM_PARENT);
```
Comment (hughsie) [inline plugins/amd-gpu/fu-amd-gpu-device.c:98]: @superm1 please can you check this, I have no AMD GPU hardware. Thanks!

Comment (superm1) [reply plugins/amd-gpu/fu-amd-gpu-device.c:98]: Here is before change:  [get-devices.txt](https://github.com/user-attachments/files/16800337/get-devices.txt)
Here is after change: [get-devices.txt](https://github.com/user-attachments/files/16800324/get-devices.txt)

Comment (superm1) [reply plugins/amd-gpu/fu-amd-gpu-device.c:98]: those are both with `fwupdtool get-devices --plugins amd-gpu -vv`

Comment (hughsie) [reply plugins/amd-gpu/fu-amd-gpu-device.c:98]: That looks identical to me -- thanks.

**Review by superm1 — APPROVED** [review]

**Pull request merged**